assets/js/tutorials/trade-in-the-dex.js

```diff
@@ -0,0 +1,278 @@
+// Source for interactive examples in the "Trade in the Decentralized Exchange"
+// tutorial.
+
+// Get Credentials, Connect steps handled by the snippet
+
+$(document).ready(() => {
+
+// Look Up Offers --------------------------------------------------------------
+
+function update_exchange_rate(event) {
+  const tpa = $("#taker-pays-amount-1").val()
+  const tga = $("#taker-gets-amount-1").val()
+  const exchange_rate = BigNumber(tga) / BigNumber(tpa)
+  $("#exchange-rate-1").val(`${exchange_rate}`)
+}
+$("#taker-pays-amount-1").change(update_exchange_rate)
+$("#taker-gets-amount-1").change(update_exchange_rate)
+
+$("#look-up-offers").click( async (event) => {
+  const block = $(event.target).closest(".interactive-block")
+  block.find(".output-area").html("")
+  const address = get_address(event)
+  if (!address) {return}
+
+  const we_want = {
+    "currency": $("#taker-pays-currency-1").val(),
+    "issuer": $("#taker-pays-issuer-1").val(),
+    "value": $("#taker-pays-amount-1").val()
+  }
+  const we_spend = {
+    "currency": "XRP",
+    "value": xrpl.xrpToDrops($("#taker-gets-amount-1").val())
+  }
+  const proposed_quality = BigNumber(we_spend.value) / BigNumber(we_want.value)
+  let orderbook_resp
+  try {
+    orderbook_resp = await api.request({
+      "command": "book_offers",
+      "taker": address,
+      "ledger_index": "current",
+      "taker_gets": we_want,
+      "taker_pays": we_spend
+    })
+    block.find(".output-area").append(
+      `<pre><code>${pretty_print(orderbook_resp.result)}`)
+  } catch(err) {
+    show_error(block, err)
+    block.find(".loader").hide()
+    return
+  }
+
+  // Estimate whether a proposed Offer would execute immediately, and...
+  // If so, how much of it? (Partial execution is possible)
+  // If not, how much liquidity is above it? (How deep in the order book would
+  //    other Offers have to go before ours would get taken?)
+  // Note: these estimates can be thrown off by rounding if the token issuer
+  // uses a TickSize setting other than the default (15). In that case, you
```

Comment:
Should this have a link to the TickSize concept page? https://xrpl.org/ticksize.html#tick-size